park: 
  {

    idle_time: 2.0,
    max_turn: 0.5,
    back_to_mid: 0.18,
    parking_speed: 0.8,
    start_parking_speed: 0.8,
    iter_distance: 0.2,
    angle_coeff: 0.5,
    turn_delay: 0.1,
    line_dist_end: 0.15
  }
